packages/next/test/unit/fixtures/02-edge-function-basepath/next.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  swcMinify: true,
```

Comment:
You can remove swcMinify since its default enabled

Reply:
Done!